### ISSUE TITLE
Move, rename and update tudatpy example applications

### ISSUE THREAD

**Issue by FilippoOggionni** [tudat-team/tudatpy-examples#1]
**Steps needed**

1. Move example applications from [here](https://github.com/tudat-team/tudatpy/tree/develop/examples) to this repository;
2. Rename examples with more descriptive names;
3. Update examples with new tudatpy module names/structure;
4. Go through examples, add a better initial description and improve inline comments.
5. Add this repository as a git submodule of tudatpy repo.

**Note**
@DominicDirkx has already updated examples 1, 2 and 10 (.py files) in tudatpy to the new setup, where examples 2 and 10 correspond to the two example applications on the website. Example 2 was renamed to "perturbed_satellite_orbit" ([here](https://tudat-space.readthedocs.io/en/latest/_src_getting_started/_src_examples/simulation_examples/delfi_simple_example_page.html) on the website), example 10 was renamed to "linear_sensitivity_analysis" ([here](https://tudat-space.readthedocs.io/en/latest/_src_getting_started/_src_examples/simulation_examples/var_equation_example_page.html) on the website).

**Comment by FilippoOggionni** [comment]
New tasks:

- [ ] delete develop branch, if necessary (@DominicDirkx could you do it if you deem necessary? I see you have merged it)
- [ ] document multibody example and add it to the tudat-space, it will be useful for Numerical Astrodynamics (do you want to do it @jo11he? We did not assign this during the meeting)

**Comment by jo11he** [comment]
I will do it, it's on my todo list

**Comment by FilippoOggionni** [comment]
@jo11he what is the status on this?

**Comment by jo11he** [comment]
Sorry, I did this a while ago but didn't update this here.

To Summarise:
I have worked over

- keplerian_satellite_orbit.py
- perturbed_satellite_orbit.py
- linear_sensitivity_analysis.py
- solar_system_propagation.py

meaning they all run on the latest tudatpy, have an "intro" ("abstract"), are properly commented (or what I consider properly) and follow PEP 8 more or less. 
Furthermore, solar_system_propagation.py is on tudat-space as a documented example showcasing multi-body propagation.

The following scripts have not been taken care of:
- earth_moon_transfer.py
- reentry_trajectory.py
because there are issues with them. If I remember there was something wrong with the guidance in  reentry_trajectory.py (Dominic was on that) and earth_moon_transfer.py didn't really make sense as a whole, not sure what the status is on that...
